setup.py

```diff
@@ -144,6 +144,10 @@ def remove_output(*sources: str) -> Generator[None, None, None]:
             stderr=sys.stderr,
         )
 
+    # pkgconf-pypi needs pybind11/share/pkgconfig to be importable
```

Comment:
FYI, directories don't need a `__init__.py` to be importable. They are called namespace packages. They're slightly cursed, but they're a thing 😅

https://packaging.python.org/en/latest/guides/packaging-namespace-packages/#native-namespace-packages

Reply:
I'm pretty sure that I tried it and it didn't work. I believe namespace packages are only a thing at the top level package, not for subpackages?

Reply:
No, they should work for any. Perhaps the `pkgconf-pypi` is broken, I was looking at it today, we'll see if I can push my changes and tests tomorrow.

Reply:
I think I've run into issues with `importlib.resources` on namespace packages, could be related?

Reply:
That would likely be because we only added support for namespace packages in `importlib.resources` in 3.10 (https://github.com/python/importlib_resources/pull/196).

It shouldn't be affecting this since we aren't using `importlib.resources`. We did a while ago, but we were using the backport for < 3.10 (it was dropped in https://github.com/pypackaging-native/pkgconf-pypi/commit/bb52fe19fd7068fe230d45662fcbafbb7559f6ce).

Reply:
I've had to add workarounds in #5598 to generate the `__init__.py` files, would be nice if I could get rid of them. :)